encoder/libav_encoder.cpp

```diff
@@ -26,14 +28,41 @@ void encoderOptionsH264M2M(VideoOptions const *options, AVCodecContext *codec)
 		codec->bit_rate = options->bitrate;
 }
 
-static const std::map<std::string, std::function<void(VideoOptions const *, AVCodecContext *)>> optionsMap =
+void encoderOptionsLibx264(VideoOptions const *options, AVCodecContext *codec)
+{
+	codec->pix_fmt = AV_PIX_FMT_YUV420P;
+
+	if (options->bitrate)
+		codec->bit_rate = options->bitrate * 1000;
```

Comment:
I thought our `options->bitrate` is bits per second (to match raspivid). Does that make `codec->bit_rate` millibits per second? Or am I hallucinating??

Reply:
> Does that make `codec->bit_rate` millibits per second? Or am I hallucinating??

Yes it does, but only for the x264 case.  This is the units it requires.